src/components/Pipelines/ParametersList/ParametersList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import pipelineParameterMapping from '../../helpers/pipelineParameterMapping';
```

Comment:
Even though I normally endorse generalization, in this case, the messages from pipeline parameter mappings are used only here. React code is already scattered among many places. I suggest you move the mappings from the helper here (unless you really thing, these messages will be used somewhere else as well).

Reply:
ok, will do